.github/workflows/unit.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
         # We should periodically check to see if another fork has taken over maintenance,
         # as the de-facto "best" fork has changed several times over the years.
         run: |
-          helm plugin install https://github.com/quintush/helm-unittest --version v0.2.11
+          helm plugin install https://github.com/helm-unittest/helm-unittest.git  --version v0.4.4
```

Comment:
Is `.git` required?
```suggestion
          helm plugin install https://github.com/helm-unittest/helm-unittest --version v0.4.4
```

Reply:
`.git` doesn't seem to be required (I tested and it works w/o) - but the `unittest` documentation https://github.com/helm-unittest/helm-unittest/tree/main?tab=readme-ov-file#install has `.git` in the install command.

Unsure if we want to remove it, or leave it as is since thats the documented install FQDN.